Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM centos:5
```

Comment:
I don't like that we're using centos:5 here. Is there any good reason for not using at least centos 6?